RocketChat/Rocket.Chat
# chore: move team mentions to ce

## Proposed changes (including videos or screenshots)
Move the Team Mentions feature to the RC Community Edition





## Issue(s)


## Steps to test or reproduce

Without the EE license, try to use the team mention feature by sending a message with `@team`, all team members should be notified.

## Further comments

TC-901
